lib/new_relic/agent/instrumentation/grape.rb

```diff
@@ -19,7 +19,8 @@
 
   depends_on do
     begin
-      if defined?(Bundler) && Bundler.rubygems.all_specs.map(&:name).include?('newrelic-grape')
+      if defined?(Bundler) && ((Bundler::VERSION >= '2' && Bundler.rubygems.installed_specs.map(&:name).include?('newrelic-grape')) ||
```

Comment:
What do you think about using `Gem::Version.new` to compare the `Bundler::VERSION` constant with the `2`?

Reply:
Made this change! https://github.com/newrelic/newrelic-ruby-agent/pull/2823/commits/b33eac4858938c8122354c9d769e6ab2c451cc61